Makefile

```diff
@@ -131,6 +131,13 @@ copy-files:
 	install -m 644 srv/salt/ceph/admin/key/*.sls $(DESTDIR)/srv/salt/ceph/admin/key/
 	install -d -m 755 $(DESTDIR)/srv/salt/ceph/admin/files
 	install -m 644 srv/salt/ceph/admin/files/*.j2 $(DESTDIR)/srv/salt/ceph/admin/files/
+	# state files apparmor
+	install -d -m 755 $(DESTDIR)/srv/salt/ceph/apparmor
+	install -d -m 755 $(DESTDIR)/srv/salt/ceph/apparmor/files
+	install -d -m 755 $(DESTDIR)/srv/salt/ceph/apparmor/files/ceph.d
+	install -m 644 srv/salt/ceph/apparmor/init.sls $(DESTDIR)/srv/salt/ceph/apparmor/
+	install -m 644 srv/salt/ceph/apparmor/files/usr* $(DESTDIR)/srv/salt/ceph/apparmor/files/
+	install -m 644 srv/salt/ceph/apparmor/files/ceph.d/* $(DESTDIR)/srv/salt/ceph/apparmor/files//ceph.d/
```

Comment:
nitpicking, there's a double `//` towards the end of this line, which is probably fine but cosmetically jarring